### DISCUSSION TITLE
questions

### DISCUSSION THREAD

**Discussion by MatejMagat305** [xoureldeen/Vectras-VM-Android#61] [Q&A]
Is it realy can use more cores? Will you add function of create qcow2? What kind of icon should I give to icon field? Thank

**Comment by xoureldeen** [comment]
1. U can use between 1 ~8 cores
2. Yes but in 2.9.1
3. The pic u want to see in Home page :)

**Reply by MatejMagat305** [reply]
You are antver me, but it is hardly believeble, because in termux I can use only one core ...
